js/packages/phoenix-mcp/src/datasetTools.ts

```diff
@@ -2,6 +2,95 @@ import { PhoenixClient } from "@arizeai/phoenix-client";
 import { McpServer } from "@modelcontextprotocol/sdk/server/mcp.js";
 import z from "zod";
 
+const LIST_DATASETS_DESCRIPTION = `Get a list of all datasets.
+
+Datasets are collections of 'dataset examples' that each example includes an input, 
+(expected) output, and optional metadata. They are primarily used as inputs for experiments.
+
+Example usage:
+  Show me all available datasets
+
+Expected return:
+  Array of dataset objects with metadata.
+  Example: [
+    {
+      "id": "RGF0YXNldDox",
```

Comment:
You may want to occlude this ID and the dates so that we don't poison the prompt